climada/engine/test/test_forecast.py

```diff
@@ -159,10 +160,11 @@ def test_Forecast_plot(self):
         map_file_name_full = Path(FORECAST_PLOT_DIR) / map_file_name
         map_file_name_full.absolute().unlink(missing_ok=False)
         forecast.plot_hist(run_datetime=dt.datetime(2017,12,31),
+                           explain_str='test text',
                            save_fig=False, close_fig=True)
         forecast.plot_exceedence_prob(run_datetime=dt.datetime(2017,12,31),
-                                      threshold=5000, save_fig=False, close_fig=True)
-
+                                      threshold=5000, explain_str='test text',
```

Comment:
Since there is quite some fiddling with titles, it might make sense to assert if the correct figure title is set with something like
```python
ax = forecast.plot_exceedence_prob(explain_str="Something from the title")
self.assertIn(ax.get_title(), "Something from the title")
```